app/services/moderation.py

```diff
@@ -125,17 +125,17 @@ def get_duration(text: str):
 
 async def user_has_now_ro(user: User, chat: Chat, bot: Bot):
     chat_member = await bot.get_chat_member(chat_id=chat.chat_id, user_id=user.tg_id)
-    if chat_member.status == ChatMemberStatus.RESTRICTED:
+    if chat_member.status == "restricted":
         assert isinstance(chat_member, ChatMemberRestricted)
         return not chat_member.can_send_messages
-    return chat_member.status in (ChatMemberStatus.BANNED, ChatMemberStatus.KICKED)
+    return chat_member.status in ("banned", "kicked")
 
 
 async def auto_restrict(target: User, chat: Chat, bot: Bot, using_db=None) -> typing.Tuple[int, ModeratorEvent]:
```

Comment:
I don't know, what's actually mean arg `using_db`, but perhaps something like that:
```suggestion
async def auto_restrict(target: User, chat: Chat, bot: Bot, using_db: typing.Optional[typing.Literal[True]] = None) -> typing.Tuple[int, ModeratorEvent]:
```

Reply:
using_db is not boolean. it something like connection for make more than one query in trasaction. fucking tortoise=(
probably we should use some other name, but it name used in orm.. I don't know=(